### PR TITLE
Service maintenance test

### PR DESCRIPTION
**Pull request by Gundersanne**
We have 2 choices really, either remove the records entirely or just zero out the manifest. This PR removes them entirely. Happy to drop this in favour of just removing the manifest however.

### CODE REVIEW

File: internal/jobqueue/dbjobqueue/dbjobqueue.go [hunk 1]
```diff
@@ -493,6 +493,9 @@ func (q *DBJobQueue) JobStatus(id uuid.UUID) (jobType string, result json.RawMes
 	var sp, fp *time.Time
 	var rp pgtype.JSON
 	err = conn.QueryRow(context.Background(), sqlQueryJobStatus, id).Scan(&jobType, &rp, &queued, &sp, &fp, &canceled)
+	if err == pgx.ErrNoRows {
```
Comment (ondrejbudai) [inline internal/jobqueue/dbjobqueue/dbjobqueue.go:496]: This should be in a separate commit I think.

**Review by ondrejbudai — COMMENTED** [review]
> If multiple osbuild jobs per depsolve are ever implemented, this
will not break.

Won't it break the data retention promise, though? If depsolve was done more than 14 days ago, the latest osbuild job based on it might be much newer/.

**Comment by Gundersanne** [comment]
> > If multiple osbuild jobs per depsolve are ever implemented, this
> > will not break.
> 
> Won't it break the data retention promise, though? If depsolve was done more than 14 days ago, the latest osbuild job based on it might be much newer/.

The assumption in this script is that the build happened within 24 hours of the depsolve. I we are working towards this, then we can't do this.

**Comment by Gundersanne** [comment]
We might need the osbuild result  around for a long time for registering purposes. For this reason I'll just open a PR which zeros out the manifest (and depsolve?) job results.
